### PR TITLE
docs: translate Legacy APIs > `createElement`

### LINKED ISSUE #549: `createElement`
Page: https://id.react.dev/reference/react/createElement

### PR DESCRIPTION
**Pull request by ilham89**
Closes #549 
## Description

Translate the Reference > Legacy APIs > createElement page.
Page URL: https://id.react.dev/reference/react/createElement

### CODE REVIEW

File: src/content/reference/react/createElement.md [hunk 1]
```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat lebih banyak contoh lainnya di bawah ini.](#usage)
 
 #### Parameters {/*parameters*/}
```
Comment (resir014) [inline src/content/reference/react/createElement.md:39]:
```suggestion
#### Parameter {/*parameters*/}
```

Use the correct translation for API reference headings.

File: src/content/reference/react/createElement.md [hunk 2]
```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
...
+* `props`: Argumen `props` harus berupa objek atau `null`. Jika Anda mengoper `null`, itu akan diperlakukan sama seperti objek kosong. React akan membuat elemen dengan *props* yang cocok dengan `props` yang telah Anda oper. Perhatikan bahwa `ref` dan `key` dari objek `props` Anda adalah spesial dan *tidak* akan tersedia sebagai `element.props.ref` dan `element.props.key` pada  `element` yang dikembalikan. Mereka akan tersedia sebagai `element.ref` dan `element.key`.
 
-* **optional** `...children`: Zero or more child nodes. They can be any React nodes, including React elements, strings, numbers, [portals](/reference/react-dom/createPortal), empty nodes (`null`, `undefined`, `true`, and `false`), and arrays of React nodes.
+* **optional** `...children`: Nol atau lebih simpul anak. Mereka bisa berupa simpul React apa saja, termasuk elemen React, string, angka, [portal](/reference/react-dom/createPortal), simpul kosong (`null`, `undefined`, `true`, dan `false`), dan array simpul React.
 
 #### Returns {/*returns*/}
```
Comment (resir014) [inline src/content/reference/react/createElement.md:47]:
```suggestion
#### Kembalian {/*returns*/}
```

Idem.

**Review by resir014 — CHANGES_REQUESTED** [review]

File: src/content/reference/react/createElement.md [hunk 3]
```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
...
 
-`createElement` returns a React element object with a few properties:
+`createElement` mengembalikan objek elemen React dengan beberapa properti:
 
-* `type`: The `type` you have passed.
-* `props`: The `props` you have passed except for `ref` and `key`. If the `type` is a component with legacy `type.defaultProps`, then any missing or undefined `props` will get the values from `type.defaultProps`.
-* `ref`: The `ref` you have passed. If missing, `null`.
-* `key`: The `key` you have passed, coerced to a string. If missing, `null`.
+* `type`: `type` yang telah Anda oper.
+* `props`: `props` yang telah Anda oper kecuali untuk `ref` dan `key`. Jika `type` adalah komponen warisan `type.defaultProps`, lalu ada yang hilang atau tidak terdefinisi `props` akan mendapatkan nilai dari `type.defaultProps`.
```
Comment (mazipan) [inline src/content/reference/react/createElement.md:52]: Warisan di sini itu dari kata legacy ya? Kayanya gak pas ya pake warisan. Kalau pakai italic aja gimana?

File: src/content/reference/react/createElement.md [hunk 4]
```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
...
+Biasanya, Anda akan mengembalikan elemen dari komponen Anda atau menjadikannya anak dari elemen lain. Meskipun Anda dapat membaca properti elemen, yang terbaik adalah memperlakukan setiap elemen sebagai buram setelah dibuat, dan hanya merendernya.
 
-#### Caveats {/*caveats*/}
+#### Peringatan {/*caveats*/}
 
-* You must **treat React elements and their props as [immutable](https://en.wikipedia.org/wiki/Immutable_object)** and never change their contents after creation. In development, React will [freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) the returned element and its `props` property shallowly to enforce this.
+* Anda harus **memperlakukan elemen React dan propertinya sebagai [kekal](https://en.wikipedia.org/wiki/Immutable_object)** dan tidak pernah mengubah isinya setelah dibuat. Dalam pengembangan, React akan [membekukan](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) elemen yang dikembalikan dan `props` properti dangkal untuk menegakkan ini.
```
Comment (mazipan) [inline src/content/reference/react/createElement.md:60]: Immutable diartikan kekal juga gak pas.
Kalau ragu, atau dirasa kurang pas, better keep as it is dulu aja.

File: src/content/reference/react/createElement.md (hunk 3 above)
Comment (ilham89) [reply src/content/reference/react/createElement.md:52]: iya legacy component, otw pake italic

File: src/content/reference/react/createElement.md (hunk 1 above)
Comment (ilham89) [reply src/content/reference/react/createElement.md]: updated

File: src/content/reference/react/createElement.md (hunk 2 above)
Comment (ilham89) [reply src/content/reference/react/createElement.md]: updated

File: src/content/reference/react/createElement.md [hunk 5]
```diff
@@ -34,44 +34,44 @@ function Greeting({ name }) {
...
+* `type`: `type` yang telah Anda oper.
+* `props`: `props` yang telah Anda oper kecuali untuk `ref` dan `key`. Jika `type` adalah komponen warisan `type.defaultProps`, lalu ada yang hilang atau tidak terdefinisi `props` akan mendapatkan nilai dari `type.defaultProps`.
+* `ref`: `ref` yang telah Anda oper. Jika hilang, `null`.
+* `key`: `key` yang telah Anda oper, dipaksa untuk string. Jika hilang, `null`.
 
-Usually, you'll return the element from your component or make it a child of another element. Although you may read the element's properties, it's best to treat every element as opaque after it's created, and only render it.
+Biasanya, Anda akan mengembalikan elemen dari komponen Anda atau menjadikannya anak dari elemen lain. Meskipun Anda dapat membaca properti elemen, yang terbaik adalah memperlakukan setiap elemen sebagai buram setelah dibuat, dan hanya merendernya.
 
-#### Caveats {/*caveats*/}
+#### Peringatan {/*caveats*/}
```
[the remark this answers is not in the document]
Comment (ilham89) [reply src/content/reference/react/createElement.md]: updated

File: src/content/reference/react/createElement.md [hunk 6]
```diff
@@ -196,10 +196,10 @@ An element is a lightweight description of a piece of the user interface. For ex
 }
 ```
 
-**Note that creating this object does not render the `Greeting` component or create any DOM elements.**
+**Perhatikan bahwa membuat objek ini tidak merender komponen `Greeting` atau membuat elemen DOM apa pun.**
 
-A React element is more like a description--an instruction for React to later render the `Greeting` component. By returning this object from your `App` component, you tell React what to do next.
+Elemen React lebih seperti deskripsi—instruksi untuk React untuk merender komponen `Greeting` nanti. Dengan mengembalikan objek ini dari komponen `App` Anda, Anda memberi tahu React apa yang harus dilakukan selanjutnya.
```
Comment (mazipan) [inline src/content/reference/react/createElement.md:201]: Semua render pake italic.
Jadi `me-*render*`

Comment (ilham89) [reply src/content/reference/react/createElement.md]: updated

**Review by resir014 — APPROVED** [review]

**Pull request merged**